packaging/fetch_gpg_key/main.go

```diff
@@ -0,0 +1,108 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// The application fetches encrypted certificate from GCS bucket,
+// decrypts the certificate using cloudkms and stores it in /tmps
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+
+	cloudkms "cloud.google.com/go/kms/apiv1"
+	"cloud.google.com/go/storage"
+	"github.com/kylelemons/godebug/pretty"
+	kmspb "google.golang.org/genproto/googleapis/cloud/kms/v1"
+)
+
+// The values are used for testing only. The values will be replaced with actual ones
```

Comment:
It's difficult to review properly with debug code, also you wont be able to edit after lgtm